engines/bastion_katello/app/assets/javascripts/bastion_katello/products/details/repositories/details/views/repository-advanced-sync.html

```diff
@@ -0,0 +1,82 @@
+<span page-title ng-model="repository">{{ 'Repository:' | translate }} {{ repository.name }}</span>
+
+<div>
+  <h3 translate>Advanced Sync</h3>
+
+  <div bst-alerts success-messages="successMessages" error-messages="errorMessages"></div>
+
+  <div class="radio">
+    <label>
+      <input type="radio" ng-model="syncType" value="standard" />
+      <span translate>
+        Standard Sync
+      </span>
+    </label>
+    <div class="" >
+      <span translate>
+        Standard sync, optimized for speed by bypassing any unneeded steps.
+      </span>
+    </div>
+  </div>
+
+  <div class="radio">
+    <label>
+      <input type="radio" ng-model="syncType" value="skipMetadataCheck" />
+      <span translate>
+        Skip Metadata Check
+      </span>
+    </label>
+    <div class="" >
+      <span translate>
+        Sync even if the upstream metadata appears to have no change.  This option will take longer than a normal sync.  Choose this option if:
+      </span>
+      <ul class="list-aligned">
+        <li translate>One or more RPMs are not showing up in the local repository even though they exist in the upstream repository.</li>
+      </ul>
+    </div>
+  </div>
+
+  <div class="radio">
+    <label>
+      <input type="radio" ng-model="syncType" value="validateContents" />
+      <span translate>
+        Validate Content
+      </span>
+    </label>
+    <div >
+      <span translate>
+        Performs a sync, verifying the checksum of all packages afterwards.  Any missing or corrupt RPMs will be re-downloaded.  This option will take the longest amount of time.  Choose this option if:
+      </span>
+      <ul class="list-aligned">
+        <li translate>Specific RPMs are throwing a 404 from yum or dnf.</li>
+        <li translate>RPMs are corrupt, often leading to an error "Package does not match intended download".</li>
+      </ul>
+      <span ng-show="repository.download_policy === 'on_demand'">
+        <i class="fa fa-warning"></i>
+        <span>
+          When validating content, repositories using the "On Demand" download policy will download all packages within the repository.
+        </span>
+      </span>
+    </div>
+  </div>
+
+  <div class="control-group buttons">
+    <button class="btn btn-primary"
+            ng-click="advancedSync(syncType); working = true;"
+            ng-disabled="syncType === undefined || working">
+
+      <i class="fa fa-spinner fa-spin" ng-show="working"></i>
+      <span ng-show="working" translate>Sync</span>
+      <span ng-hide="working" translate>Sync</span>
```

Comment:
This seems redundant, show the word "Sync" on `working` and also hide it on `working`?